CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Fix `onlyVisibleForYouIndicator` not being shown for ephemeral messages [#1948](https://github.com/GetStream/stream-chat-swift/pull/1948)
 - Channel not being marked as read by the current user when it's only thread replies or silent messages sent after the last read [#1904](https://github.com/GetStream/stream-chat-swift/issues/1904)
+- Fix footnote being hidden for the message followed by `ephemeral` message [#1956](https://github.com/GetStream/stream-chat-swift/issues/1956)
+- Fix footnote being hidden for the message followed by `system` message [#1956](https://github.com/GetStream/stream-chat-swift/issues/1956)
```

Comment:
Since this PR no longer affects delivery status and has only bug fixes, I re-target it to point to the `develop` branch. Having changelog entries for those fixes now makes sense